airbyte-integrations/connectors/source-stripe/integration_tests/expected_records.jsonl

```diff
@@ -49,7 +49,6 @@
oupon":{"id":"4SUEGKZg","object":"coupon","amount_off":200.0,"created":1674209030,"currency":"usd","duration":"repeating","duration_in_months":3.0,"livemode":false,"max_redemptions":null,"metadata":{},"name":"Іекшзу","percent_off":null,"redeem_by":null,"times_redeemed":2.0,"valid":true},"customer":"cus_Kou8knsO3qQOwU","end":1725547362,"invoice":null,"invoice_item":null,"promotion_code":null,"start":1717598562,"subscription":"sub_1POL30EcXtiJtvvh4UdCl3kj","subscription_item":null},"discounts":["di_1POL31EcXtiJtvvhc8sEBzwi"],"ended_at":null,"invoice_settings":{"account_tax_ids":null,"issuer":{"type":"self"}},"items":{"object":"list","data":[{"id":"si_QEogVBC6jV71Ti","object":"subscription_item","billing_thresholds":null,"created":1717598563,"discounts":[],"metadata":{},"plan":{"id":"price_1PDtzkEcXtiJtvvhY4aqjMSq","object":"plan","active":true,"aggregate_usage":null,"amount":null,"amount_decimal":null,"billing_scheme":"tiered","created":1715111292,"currency":"usd","interval":"week","interval_count":1,"livemode":false,"metadata":{},"meter":null,"nickname":null,"product":"prod_Q423JP7oX7TF5U","tiers_mode":"graduated","transform_usage":null,"trial_period_days":null,"usage_type":"licensed"},"price":{"id":"price_1PDtzkEcXtiJtvvhY4aqjMSq","object":"price","active":true,"billing_scheme":"tiered","created":1715111292,"currency":"usd","custom_unit_amount":null,"livemode":false,"lookup_key":null,"metadata":{},"nickname":null,"product":"prod_Q423JP7oX7TF5U","recurring":{"aggregate_usage":null,"interval":"week","interval_count":1,"meter":null,"trial_period_days":null,"usage_type":"licensed"},"tax_behavior":"exclusive","tiers_mode":"graduated","transform_quantity":null,"type":"recurring","unit_amount":null,"unit_amount_decimal":null},"quantity":1,"subscription":"sub_1POL30EcXtiJtvvh4UdCl3kj","tax_rates":[]}],"has_more":false,"total_count":1.0,"url":"/v1/subscription_items?subscription=sub_1POL30EcXtiJtvvh4UdCl3kj"},"latest_invoice":"in_1PYUP7EcXtiJtvvhSPc5z2FG","livemode":false,"metadata":{},"next_pending_invoice_item_invoice":null,"on_behalf_of":null,"pause_collection":null,"payment_settings":{"payment_method_options":null,"payment_method_types":null,"save_default_payment_method":"off"},"pending_invoice_item_interval":null,"pending_setup_intent":null,"pending_update":null,"plan":{"id":"price_1PDtzkEcXtiJtvvhY4aqjMSq","object":"plan","active":true,"aggregate_usage":null,"amount":null,"amount_decimal":null,"billing_scheme":"tiered","created":1715111292,"currency":"usd","interval":"week","interval_count":1,"livemode":false,"metadata":{},"meter":null,"nickname":null,"product":"prod_Q423JP7oX7TF5U","tiers_mode":"graduated","transform_usage":null,"trial_period_days":null,"usage_type":"licensed"},"quantity":1,"schedule":null,"start_date":1717598562,"status":"past_due","test_clock":null,"transfer_data":null,"trial_end":null,"trial_settings":{"end_behavior":{"missing_payment_method":"create_invoice"}},"trial_start":null,"updated":1717598562},"emitted_at":1720515564910}
```

Comment:
i have no idea why this record got removed from our stripe account, but I couldn't retrieve it listing subscriptions via http requests and trying the direct request using the `id` using `https://api.stripe.com/v1/subscriptions/sub_1MowQVLkdIwHu7ixeRlqHVzs`, resulted in a 404, so I don't believe it exists anymore.

This was passing last night/this morning so i'm convinced this isn't an issue with my changes